pkg/deploy/controller/configchange/controller.go

```diff
@@ -27,7 +27,7 @@ type DeploymentConfigChangeController struct {
 // fatalError is an error which can't be retried.
 type fatalError string
 
-func (e fatalError) Error() string { return "fatal error handling config: " + string(e) }
+func (e fatalError) Error() string { return "fatal error handling configuration: " + string(e) }
```

Comment:
Now that I'm thinking of @soltysh's suggestion, string concatenation ("string" + "other string") is more expensive and you slightly loose the formatting control you get with `fmt.Sprintf` so I would use that instead.